api/api.go

```diff
@@ -101,3 +101,70 @@ func OrbQuery(ctx context.Context, logger *logger.Logger, configPath string) (*C
 			}
 		}`)
 }
+
+type CreateNamespaceResponse struct {
+	Namespace struct {
+		CreatedAt string
+		Id string
+	}
+
+	Errors []struct {
+		Message string
+		Type string
+	}
+}
+
+func (response CreateNamespaceResponse) ToError() error {
+	messages := []string{}
+
+	for i := range response.Errors {
+		messages = append(messages, response.Errors[i].Message)
```

Comment:
Looks like you are not using the `type` field here. You should either use it, or drop it from the GQL request.

Reply:
✅